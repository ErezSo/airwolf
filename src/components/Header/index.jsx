import React from "react";
import { Cart } from "../";

const styles = {
  container: {
    width: "100%",
    display: "flex",
    height: 100,
    alignItems: "center",
    justifyContent: "space-between"
  },
  left: {
    margin: "0 40px"
  },
  cart: {
    margin: "0 40px"
  }
};

export default function Header({ cartCounter }) {
  const { container, left, cart } = styles;
  return (
    <div style={container}>
      <span style={left} />
      <span>
        <img src="assets/vimcar-logo.svg" alt="logo" />
      </span>
      <span style={cart}>
        <Cart counter={cartCounter} />
      </span>
    </div>
  );
}
