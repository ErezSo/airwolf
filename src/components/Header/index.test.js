import React from "react";
import { shallow } from "enzyme";
import Header from "./";

describe("Header", () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<Header />);
  });

  it("renders", () => {
    expect(wrapper).toHaveLength(1);
  });

  it("renders correctly", () => {
    expect(wrapper.debug()).toMatchSnapshot();
  });
});
