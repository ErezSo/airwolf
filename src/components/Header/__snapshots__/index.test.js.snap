// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders correctly 1`] = `
"<div style={{...}}>
  <span style={{...}} />
  <span>
    <img src=\\"assets/vimcar-logo.svg\\" alt=\\"logo\\" />
  </span>
  <span style={{...}}>
    <Cart counter={[undefined]} />
  </span>
</div>"
`;
